library/alloc/src/raw_vec.rs

```diff
@@ -186,34 +203,37 @@ impl<T, A: Allocator> RawVec<T, A> {
 
     #[cfg(not(no_global_oom_handling))]
     fn allocate_in(capacity: usize, init: AllocInit, alloc: A) -> Self {
+        match Self::try_allocate_in(capacity, init, alloc) {
+            Ok(r) => r,
+            Err(e) => e.handle(),
+        }
```

Comment:
I saw some comment about `unwrap_or_else` slowing doing the LLVM portion of compilation so I didn't use it, but if that is no longer the case I am happy to change these.